source/jormungandr/tests/instant_system_new_default_routing_tests.py

```diff
@@ -0,0 +1,223 @@
+        assert ridesharing_kraken.get('type') == 'best'
+        assert ridesharing_kraken.get('durations').get('ridesharing') == 13
+        assert ridesharing_kraken.get('durations').get('total') == 13
+
+        rs_sections = ridesharing_kraken.get('sections')
+        assert len(rs_sections) == 1
+        assert rs_sections[0].get('mode') == 'ridesharing'
+        assert rs_sections[0].get('type') == 'street_network'
+
+        ridesharing = journeys[1]
+        assert 'ridesharing' in ridesharing['tags']
+        assert 'ecologic' in ridesharing['tags']
+        assert ridesharing.get('type') == 'rapid'
+
+        distances = ridesharing.get('distances')
+        assert distances.get('ridesharing') == 211
+        assert distances.get('car') == 0
+        assert distances.get('walking') == 0
+
+        durations = ridesharing.get('durations')
+        assert durations.get('ridesharing') == 926
+        assert durations.get('total') == 926
+
+        rs_sections = ridesharing.get('sections')
+        assert len(rs_sections) == 1
+        assert rs_sections[0].get('mode') == 'ridesharing'
+        assert rs_sections[0].get('type') == 'crow_fly'
+
+        rs_journeys = rs_sections[0].get('ridesharing_journeys')
+        assert len(rs_journeys) == 1
+        assert rs_journeys[0].get('distances').get('walking') == 16
+        assert rs_journeys[0].get('distances').get('ridesharing') == 224
+        assert rs_journeys[0].get('durations').get('walking') == 0 #two crow_fly sections have 0 duration
+        assert rs_journeys[0].get('durations').get('ridesharing') == 1057
+        assert 'ridesharing' in rs_journeys[0].get('tags')
+        rsj_sections = rs_journeys[0].get('sections')
+        assert len(rsj_sections) == 3
+
+        assert rsj_sections[0].get('type') == 'crow_fly'
+        assert rsj_sections[0].get('mode') == 'walking'
+
+        assert rsj_sections[1].get('type') == 'ridesharing'
+        rsj_info = rsj_sections[1].get('ridesharing_informations')
+        assert rsj_info.get('driver').get('alias') == 'Jean P.'
+        assert rsj_info.get('driver').get('gender') == 'male'
+        assert rsj_info.get('driver').get('image') == 'https://dummyimage.com/128x128/C8E6C9/000.png&text=JP'
+        assert rsj_info.get('driver').get('rating').get('scale_min') == 0.0
+        assert rsj_info.get('driver').get('rating').get('scale_max') == 5.0
+        assert rsj_info.get('network') == 'Super Covoit'
+        assert rsj_info.get('operator') == 'Instant System'
+        assert rsj_info.get('seats').get('available') == 4
+        assert rsj_info.get('seats').get('total') == 4
+
+        rsj_links = rsj_sections[1].get('links')
+        assert len(rsj_links) == 2
+        assert rsj_links[0].get('rel') == 'ridesharing_ad'
+        assert rsj_links[0].get('type') == 'ridesharing_ad'
+
+        assert rsj_links[1].get('rel') == 'tickets'
+        assert rsj_links[1].get('type') == 'ticket'
+        assert rsj_links[1].get('id') == ticket_id
```

Comment:
missing tests:
* on tickets (check that link to the section is OK)
* on the prices (ridesharing_journey AND ticket itself)